lib/at_cmd_parser/at_utils.h

```diff
@@ -20,11 +20,13 @@
 #include <ctype.h>
 
 #define AT_PARAM_SEPARATOR              ','
-#define AT_CMD_SEPARATOR                ':'
+#define AT_RSP_SEPARATOR                ':'
+#define AT_CMD_SEPARATOR                '='
 #define AT_CMD_BUFFER_TERMINATOR        0
 #define AT_CMD_STRING_IDENTIFIER        '\"'
 #define AT_STANDARD_NOTIFICATION_PREFIX '+'
 #define AT_PROP_NOTIFICATION_PREFX      '%'
+#define AT_CUSTOM_COMMAND_PREFX         '#'
```

Comment:
Like '%' we also use '#' as custom notification prefix. Maybe we could name it AT_CUST_NOTIFICATION_PREFX just to align with proprietary commands on modem side.